### DISCUSSION TITLE
[Nhận task]

### DISCUSSION THREAD

**Discussion by trungdoublelift** [Runway-Club/Runway-CV-Builder#95] [Task Delivery]
Tôi muốn  nhận task: Chức năng xuất donwnload CV về máy

**Comment by nguyentantrung2000** [comment]
Thông tin task:
Cho người dùng có thể download cv của chính mình về máy 
Yêu cầu:
Chức năng export pdf
Deadline 11h59 20/8/2021

**Reply by trungdoublelift** [reply]
OK
